hackathon_site/event/api_urls.py

```diff
@@ -13,5 +13,10 @@
         name="join-team",
     ),
     path("teams/leave_team/", api_views.LeaveTeamView.as_view(), name="leave-team"),
+    re_path(
+        r"^profiles/(?P<id>[0-9]+)/$",
```

Comment:
It's usually a convention to do `re_path("profiles/<int:pk>/`. `pk` (primary key) is the default lookup argument for django, which will end up doing an `id` lookup in the database. Saves you from having to set the `lookup_field` in the view